### ISSUE TITLE
NRK login

### ISSUE THREAD

**Issue by rytmisk** [retrospect-addon/plugin.video.retrospect#1669]
### Is your feature request related to a problem? Please describe

Hi
Retrospect is awesome! I use it all the time. One thing I do miss is the opportunity to log in to NRK. The reason why that would be awesome is mainly because it allows NRK to play even outside Norway if you are a registered citizen. For someone like me who lives abroad, that is fantastic. It works in the android NRK app, but that app is almost useless on my "smart"-tv.
Besides that - the login provides possibilities to track what is being watched etc. 

All the best!
Ketil

**Comment by rytmisk** [comment]
Any news regarding this?
All the best!
Ketil

**Comment by basrieter** [comment]
No, I am very sorry. No progress here. Log-ins generally are difficult to implement and I just did not get to it.

**Comment by rytmisk** [comment]
No worries! You are still the best!
